runtime/vm/visible.c

```diff
@@ -296,6 +296,18 @@ loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
 			nestHostName = J9ROMCLASS_NESTHOSTNAME(curClazz->romClass);
 		}
 
+		if (hiddenNestMate) {
```

Comment:
Can you move this `if` earlier to before the while loop?  It makes it clearer this code only applies to hidden classes.

Reply:
Done.

Reply:
Also moved variable `isCurClassHiddenNestMate` inside to the `if` block.